pom.xml

```diff
@@ -265,6 +250,13 @@
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <!-- Avoid it from being bundled -->
```

Comment:
Ditto, where was this coming from before?

Reply:
Compile scope comes this trail

```
+- org.jenkinsci.plugins:kubernetes-credentials:jar:0.6.2:compile
|  +- org.jenkins-ci.plugins:apache-httpcomponents-client-4-api:jar:4.5.10-2.0:compile
|  |  +- org.apache.httpcomponents:httpclient:jar:4.5.10:compile
|  |  |  \- org.apache.httpcomponents:httpcore:jar:4.4.12:compile
|  |  +- org.apache.httpcomponents:httpmime:jar:4.5.10:compile
|  |  +- org.apache.httpcomponents:fluent-hc:jar:4.5.10:compile
|  |  +- org.apache.httpcomponents:httpclient-cache:jar:4.5.10:compile
|  |  +- org.apache.httpcomponents:httpasyncclient:jar:4.1.4:compile
|  |  |  \- org.apache.httpcomponents:httpcore-nio:jar:4.4.10:compile
|  |  \- org.apache.httpcomponents:httpasyncclient-cache:jar:4.1.4:compile
|  \- org.jenkins-ci.plugins:google-oauth-plugin:jar:0.8:compile
|     +- joda-time:joda-time:jar:2.9.5:compile
|     +- org.jenkins-ci.plugins:oauth-credentials:jar:0.3:compile
|     |  \- net.sourceforge.findbugs:jsr305:jar:1.3.2:compile
```

Reply:
Trails
```
+- org.jenkinsci.plugins:kubernetes-credentials:jar:0.6.2:compile
|  +- org.jenkins-ci.plugins:apache-httpcomponents-client-4-api:jar:4.5.10-2.0:compile
|  |  +- org.apache.httpcomponents:httpclient:jar:4.5.10:compile
|  |  |  \- org.apache.httpcomponents:httpcore:jar:4.4.12:compile
|  |  +- org.apache.httpcomponents:httpmime:jar:4.5.10:compile
|  |  +- org.apache.httpcomponents:fluent-hc:jar:4.5.10:compile
|  |  +- org.apache.httpcomponents:httpclient-cache:jar:4.5.10:compile
|  |  +- org.apache.httpcomponents:httpasyncclient:jar:4.1.4:compile
|  |  |  \- org.apache.httpcomponents:httpcore-nio:jar:4.4.10:compile
|  |  \- org.apache.httpcomponents:httpasyncclient-cache:jar:4.1.4:compile
|  \- org.jenkins-ci.plugins:google-oauth-plugin:jar:0.8:compile
|     +- joda-time:joda-time:jar:2.9.5:compile
|     +- org.jenkins-ci.plugins:oauth-credentials:jar:0.3:compile
|     |  \- net.sourceforge.findbugs:jsr305:jar:1.3.2:compile
|     +- com.google.http-client:google-http-client:jar:1.24.1:compile
|     |  \- com.google.code.findbugs:jsr305:jar:3.0.2:compile
```

```
+- com.github.spotbugs:spotbugs-annotations:jar:3.1.12:provided (optional) 
|  \- com.google.code.findbugs:jsr305:jar:3.0.2:compile
```

The one from spotbugs-annotations is much shorter, so this is the one that gets picked up during packaging.